gui/src/routes.ts

```diff
@@ -18,7 +20,9 @@ export default {
   '/workspaces/:workspace/components': WorkspaceComponents,
   '/workspaces/:workspace/storage': WorkspaceStorage,
   '/workspaces/:workspace/networking': WorkspaceNetworking,
+  '/workspaces/:workspace/new-component': WorkspaceNewComponent,
   '/workspaces/:workspace/new-process': NewProcess,
+  '/workspaces/:workspace/new-docker-container': NewDockerContainer,
```

Comment:
let's just call this "new-container" and drop the word "docker". We may still want to refer to the category of things as "docker" or the provider as such, but as a general rule containers etc are outgrowing docker-the-brand.